L1Trigger/L1CaloTrigger/test/test_Phase1L1TJets_cfg.py

```diff
@@ -29,23 +29,23 @@
 
 # AK4 PF jets
 process.load('L1Trigger.Phase2L1ParticleFlow.l1pfJetMet_cff')
-process.l1PFJets = cms.Sequence( process.ak4PFL1Puppi + process.ak4PFL1PuppiCorrected )
+process.L1TPFJets = cms.Sequence( process.ak4PFL1Puppi + process.ak4PFL1PuppiCorrected )
 
 
 process.out = cms.OutputModule("PoolOutputModule",
   fileName = cms.untracked.string('myOutputFile.root'),
   outputCommands = cms.untracked.vstring(
     "drop *",
-    "keep *_Phase1L1TJetProducer*_*_*",
-    "keep *_Phase1L1TJetSumsProducer*_*_*",
+    "keep *_l1tPhase1JetProducer*_*_*",
+    "keep *_l1tPhase1JetSumsProducer*_*_*",
     "keep *_ak4GenJetsNoNu_*_*",
-    "keep *_Phase1L1TJetCalibrator*_*_*",
+    "keep *_l1tPhase1JetCalibrator*_*_*",
     "keep *_ak4PFL1Puppi*_*_*",
     "keep *_l1PFMetPuppi*_*_*",
```

Comment:
Shouldn't these 2 follow the `l1t` naming convention?

Reply:
The cfg was removed as it was calling old deprecated modules